### DISCUSSION TITLE
Nice mod

### DISCUSSION THREAD

**Discussion by Cheolhyeol** [Felicis/Water-Physics-Overhaul#57] [General]
I really liked this mod and wanted to use it in a 1.20.1 modpack, would it be possible to update it to 1.20.1?
and an idea for the mod is that hot blocks evaporate water, so when it rains the amount of water will be the same as at the beginning

**Comment by Stikulzon** [comment]
https://github.com/Stikulzon/Water-Physics-Overhaul

**Reply by Cheolhyeol** [reply]
tyy, but how can i get .jar file? idk how to use github

**Reply by Cheolhyeol** [reply]
my game crashed when i use my modpack
![Captura de tela 2024-07-04 120249](https://github.com/Felicis/Water-Physics-Overhaul/assets/163346019/34e1c97f-16ee-47fe-9072-32e6fb60ac1e)
![Captura de tela 2024-07-04 120440](https://github.com/Felicis/Water-Physics-Overhaul/assets/163346019/2df8e6aa-3fd6-486c-b60f-d1118927723c)

**Reply by Stikulzon** [reply]
1 - TLauncer, maybe you want something like SKLauncher.
2 - Send full log

**Reply by Cheolhyeol** [reply]
[crash-2024-07-08_14.56.03-server.txt](https://github.com/user-attachments/files/16132828/crash-2024-07-08_14.56.03-server.txt)
